ui/src/utils/deepFlattenObj.js

```diff
@@ -0,0 +1,39 @@
+import { camelCase } from 'lodash';
+
+/**
+ * Creates a flat object from a nested object
+ * Note: this implementation does not account for the possibility
+ * of having nested keys that are the same, one would clober the other
+ * but for our current application key brevity is the priority
+ * (see function below for key safe alternative)
+ * @param {Object} object - object to flatten
+ * @returns {Object} - flat object containing all nested keys with values
+ */
+export default function deepFlattenObjNaive(object) {
+  return Object.keys(object).reduce((acc, curr) => {
+    if (typeof object[curr] == 'object') {
+      return Object.assign(deepFlattenObj(object[curr]), acc);
+    } else {
+      acc[curr] = object[curr];
```

Comment:
should the key here also be `camelCase`d not sure but I see this mix of camel case and snake in the tsv:

`clinicalTumorDiagnosis | aquisitionSite | histologicalType | histologcalGrade | clinicalStageGrouping | source_model_url | source_sequence_url | name | type | split_ratio | growth_rate | primary_site | neoadjuvant_therapy | tnm_stage | molecular_characterizations | age_at_diagnosis | vital_status | gender | race | chemotherapeutic_drug_list_available | age_at_aquisition | disease_status | therapy | licensing_required | date_of_availability`